kubernetes-elasticsearch-cluster/es-client.yaml

```diff
@@ -43,7 +43,7 @@ spec:
         - name: HTTP_ENABLE
           value: "true"
         - name: ES_JAVA_OPTS
-          value: -Xms256m -Xmx256m
+          value: -Xms1024m -Xmx1024m
```

Comment:
After this round of changes, please manually test that this works with n1-standard-4 cluster.

Reply:
Works.